include/openssl/x509.h

```diff
@@ -2997,6 +3002,19 @@ OPENSSL_EXPORT void X509_STORE_CTX_set0_param(X509_STORE_CTX *ctx,
 OPENSSL_EXPORT int X509_STORE_CTX_set_default(X509_STORE_CTX *ctx,
                                               const char *name);
 
+// X509_STORE_get_by_subject is an alias to |X509_STORE_CTX_get_by_subject| in
+// OpenSSL 1.1.1.
+#define X509_STORE_get_by_subject X509_STORE_CTX_get_by_subject
```

Comment:
Does this cause an issue with `X509_STORE_get_by_subject` being exported up on line 2909 as a symbol but it's just a macro?

Reply:
Oh man, great catch I just forgot about removing the original symbol when I discovered it was changed to an alias.
For the record, I think subsequent symbols defined at the bottom of the file take priority over the ones at the top, so this would cause an overwrite.